lib/ts/fetch.ts

```diff
@@ -573,29 +584,8 @@ type IdRefreshTokenType =
 // else we return undefined.
 export async function getIdRefreshToken(tryRefresh: boolean): Promise<IdRefreshTokenType> {
     logDebugMessage("getIdRefreshToken: called");
```

Comment:
tryRefresh is false:
   - if front token exists -> EXISTS
   - if front token does not exist, lastRefreshTime exists -> NOT_EXISTS
   - Else -> MAY_EXISTS

try refresh is true:
   - if front token exists -> EXISTS
   - if front token does not exist, lastRefreshTime exists -> NOT_EXISTS
   - Else -> We try refreshing the session